### DISCUSSION TITLE
fastcdr2.x version not supported，help.

### DISCUSSION THREAD

**Discussion by Tooo-Top** [eProsima/RPC#26] [Ideas]
src/protocols/dds/MessageHeader.cpp,Some functions in fastcdr2. x have changed. Can you update them.

**Comment by EduPonz** [comment]
I'm going to move this ticket to the appropriate forum, as it is more a request than a feature. We will upgrade to Fast CDR v2 in the near future, but for now you can use RPC in combination with the latest Fast DDS and Fast CDR v1.

**Reply by Tooo-Top** [reply]
be waiting to see

**Reply by EduPonz** [reply]
Just FYI:

* #28
